sql/src/main/java/io/crate/analyze/AnalyzedColumnDefinition.java

```diff
@@ -146,6 +147,11 @@ void objectType(String objectType) {
         this.objectType = objectType;
     }
 
+    @VisibleForTesting
+    String getObjectType() {
```

Comment:
maybe it makes sense to make the variable testing visible instead of creating a getter which is only used in tests